src/java.base/share/classes/java/util/random/package-info.java

```diff
@@ -125,8 +125,8 @@
  *
  * <h2>Choosing a Random Number Generator Algorithm</h2>
  *
- * <p> There are three groups of random number generator algorithm provided
- * in Java: the Legacy group, the LXM group, and the Xoroshiro/Xoshiro group.
+ * <p> Random number generator algorithms are organized in groups,
+ * as described <a href="package-summary.html#algorithms">below</a>.
```

Comment:
FYI, it is possible to link to an anchor in a package-info file using syntax like

`@link java.util.random##algorithms`